### PR TITLE
change IP host

### PR DESCRIPTION
**Pull request by enoch85**

### CODE REVIEW

File: lib.sh [hunk 1]
```diff
@@ -38,22 +37,19 @@ KEYBOARD_LAYOUT=$(localectl status | grep "Layout" | awk '{print $3}')
 # HYPERVISOR=$(dmesg --notime | grep -i hypervisor | cut -d ':' -f2 | head -1 | tr -d ' ') TODO
 SYSVENDOR=$(cat /sys/devices/virtual/dmi/id/sys_vendor)
 # Network
-first_iface() {
```
Comment (szaimen) [inline lib.sh:41]: if you remove this function you might also remove it everywhere where it is used...

Comment (enoch85) [reply lib.sh:41]: This is the official VM, it's a bit different.

Need to double check this though.

Comment (szaimen) [reply lib.sh:41]: ah okay

Comment (szaimen) [reply lib.sh:41]: I didn't see that it is going to be merged into the official vm ;)

Comment (enoch85) [reply lib.sh:41]: In the official VM the lib isn't renewed every time, it stays local, and is released with every release.

So this should work.

**Pull request merged**